peeps.json

```diff
@@ -11,6 +11,10 @@
     {
       "names": ["Dustin"],
       "location": "Nanaimo"
+    },
+    {
+      "names": ["Melissa"],
+      "location": "Kimberley, BC, Canada"
```

Comment:
The weather service that I use puts you in South Africa with that address. I tried using `V1A` and it says Cranbrook.